src/components/App/index.tsx

```diff
@@ -69,8 +73,16 @@ export const App: FC = () => {
   if (loading || loadingW) return <Loader />;
   if (errorW) return <ErrorModal />;
 
+  const onClickAppHandler = (e: React.MouseEvent<HTMLDivElement>) => {
+    if (!(e.target as HTMLElement).closest('.CommonSelectList')) {
```

Comment:
This is a click outside handler, right? I'd recommend to move it to `Select` component.

Reply:
Done